src/Uuid/UuidGenerator.php

```diff
@@ -33,12 +33,12 @@ public function generateV4() {
      *
      * @return String   Valid v5 UUID.
      */
-    public function generateV5($str, $namespace = NULL) {
+    public function generateV5($namespace = NULL) {
         // Use default namespace if none is provided.
         if (empty($namespace)) {
             $namespace = $this->namespace;
         }
 
-        return Uuid::uuid5($namespace, $str)->toString();
+        return Uuid::uuid5(Uuid::NAMESPACE_DNS, $namespace)->toString();
```

Comment:
We do allow it. The ```$namespace``` variable is passed in. But based on the examples here (https://github.com/ramsey/uuid) it appears that we don't pass in a string
```
// Generate a version 5 (name-based and hashed with SHA1) UUID object
$uuid5 = Uuid::uuid5(Uuid::NAMESPACE_DNS, 'php.net');
echo $uuid5->toString() . "\n"; // i.e. c4a760a8-dbcf-5254-a0d9-6a4474bd1b62